model/serialized_gen.go

```diff
@@ -0,0 +1,1622 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
A lot of generated code here in the same model package. I know we are adding methods to the same struct, but my concern is generating too much code in the same package and make the compilation process of the `model` package slower because of that (no compile cache).

Reply:
That's a fair concern. I don't see a way around that unfortunately. The methods need to be of the same struct. And also arguably the `model` layer is modified much less often than app or api4.